WordPress/build.gradle

```diff
@@ -478,6 +478,7 @@ dependencies {
     testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$gradle.ext.kotlinVersion"
     testImplementation "org.assertj:assertj-core:$assertjVersion"
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$kotlinxCoroutinesVersion"
+    testImplementation "org.jetbrains.kotlin:kotlin-reflect:$gradle.ext.kotlinVersion"
```

Comment:
Why is this necessary? Tests are still passing when I remove it.

Reply:
I was having trouble with a test, and I needed it. I think I went a different route and you are probably right. I can remove it.